README.md

```diff
@@ -144,17 +164,18 @@ enable heuristics to make this invisible in most cases, but will not fix it comp
 
 ## Security & Privacy considerations
 
-This should be under the same restrictions as the `navigator.clipboard.read()`:
+This in of itself does not provide the website with any new substantial information about the user. The only potential
+danger is a new fingerprinting surface. To remediate this:
 
-- It should require `clipboard-read` permissions and request them on call.
-- It should be available only while the tab has focus.
+- This should be available only when the document is in focus (same as `navigator.clipboard.read()`).
+- For user agents implementing persistent clipboard permissions, it might make sense to put this behind `clipboard-read`
+  permission - as there should be no application of this that won't require calling `read()` sometime later.
```

Comment:
I think Chrome could imagine offering read() without permission if it happens while handling ctrl-v, in which case contentsId() would benefit from also not requiring permission. So I'd drop this bullet.